### PR TITLE
fix problems discovered during BK, and as mockserver was being experimented with.

### PR DESCRIPTION
**Pull request by julialongtin**

### CODE REVIEW

File: docker-squid/mnt/squid.conf [hunk 1]
```diff
@@ -66,3 +66,11 @@ http_access allow all
 # - http://www.squid-cache.org/Doc/config/cache_peer
 # - https://www.sbarjatiya.com/notes_wiki/index.php/Cache_peer_configuration_for_squid
 #cache_peer sib1.foo.net parent 3128 0 proxy-only no-query
+
+offline_mode off
```
Comment (fisx) [inline docker-squid/mnt/squid.conf]: add a comment:

```
# there is little evidence this improves offline behavior.  this is the default. we just make it explicit here.
```

File: docker-squid/run.sh [hunk 2]
```diff
@@ -6,11 +6,13 @@ IMG_TAG=quay.io/wire/squid@sha256:0df70cbcd1faa7876e89d65d215d86e1518cc45e24c7bf
...
 
-NETWORKING="--dns=8.8.8.8 --network=host"
+NETWORKING="--network=host"
 #NETWORKING="-p 3128:3128 -p 3129:3129"
 
 SETUP_TLS="
-    -v /etc/ssl/certs:/etc/ssl/certs:ro"
+    -v /etc/ssl/certs:/etc/ssl/certs:ro
+    -v /usr/share/ca-certificates:/usr/share/ca-certificates:ro
+    -v /usr/local/share/ca-certificates:/usr/local/share/ca-certificates:ro"
```
Comment (fisx) [inline docker-squid/run.sh]: I wonder if that's redundant?  Is /etc/ssl/certs fed by /usr/share/.../certificates?

Not important, better 2 than 0.  :)

File: docker-squid/mnt/squid.conf [hunk 3]
```diff
@@ -8,10 +8,16 @@ logfile_rotate 0
...
-access_log daemon:/mnt/log/access.log squid
+access_log daemon:/mnt/log/access.log mockserver
 cache_store_log daemon:/mnt/log/store.log
 cache_log /mnt/log/cache.log
 
+# log formats:
+## The default. listed here for comparison
+logformat squid      %ts.%03tu %6tr %>a %Ss/%03>Hs %<st %rm %ru %[un %Sh/%<a %mt
+## A preferred one, developed during mockserver testing.
+logformat mockserver      %ts.%03tu %6tr %>a %Ss/%<Hs/%03>Hs %<st %ru %<a
```
Comment (fisx) [inline docker-squid/mnt/squid.conf]: add a link to mockserver?  this may be the wrong place, not sure.

File: docker-squid/run.sh (hunk 2 above)
Comment (julialongtin) [reply docker-squid/run.sh]: it's not redundant. by default, /etc/ssl/certs creates symlinks into the other directories. hense, some of the SSL problems we've had.

**Pull request merged**